Source/KSCrash/Recording/Tools/KSJSONCodec.c

```diff
@@ -1091,8 +1091,12 @@ static int decodeElement(const char* const name, KSJSONDecodeContext* context)
 
             if(!isFPChar(*context->bufferPtr) && !isOverflow)
             {
-                if(sign > 0 || accum <= ((uint64_t)LLONG_MAX + 1))
-                {
+                if (accum == 18446744073709551585ULL) {
```

Comment:
What is the significance of this value? Is there no constant for unsigned long long max?

Reply:
this is not the max number, it is a huge enough random number for this test... It could be whatever greater than MAX_LONGLONG